tests/unit/conftest.py

```diff
@@ -10,3 +10,32 @@ def root_dir(request):
 @pytest.fixture(scope="module")
 def local_env_file(root_dir):
     yield root_dir / "tests" / "test.env"
+
+
+@pytest.fixture(scope="session")
+def session_plex():
+    """
+    Fixture to return the prometheus log exporter module.
+    """
+    import sys
+    from pathlib import Path
+    import importlib.util
+
+    # Ensure the parent directory is in the sys.path
+    root = Path(__file__).parent.parent.parent
+    path = root  / "birdhouse" / "optional-components" / "prometheus-log-parser" / "config" / "thredds"
+    spec = importlib.util.spec_from_file_location("prometheus_log_exporter", str(path / "prometheus-log-exporter.py"))
```

Comment:
This method of importing from a file only approximates the actual import behaviour and is not recommended in most cases (see: https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly). It does work here but there are much simpler methods that are better suited to this use-case. 

Here are two options including how to use them:

```python
import runpy
mod_namespace = runpy.run_path(str(path / "prometheus-log-exporter.py"))
mod_namespace["parse_line"]("some log line here")
```

```python
import sys
import importlib
sys.path.append(str(path))
mod = importlib.import_module("prometheus-log-exporter")
mod.parse_line("some log line here")
```

This second one is probably the one you want since it's the closest you'll get to the behaviour of a real import

Reply:
The second option is what I tried at first, but I remember I had problems with it, possibly due to the root_dir issue you mentionned.